onnxruntime/core/common/make_unique.h

```diff
@@ -0,0 +1,148 @@
+// Copyright 2017 The Abseil Authors.
```

Comment:
Do we need something listing the github repo where the file came from? 

Also is there a list of OSS used by ORT that needs abseil added to it?

Reply:
yes, was in the process of doing so while you were writing this :). I've added the github link in the TPN file. Can add here as well. Doesn't hurt.